packages/bng-metric-service/src/helpers/extractors/bng-metric-single-data-extractor.js

```diff
@@ -110,30 +110,24 @@ class BngMetricSingleDataExtractor {
 
   #removeUnwantedColumns = (data, extractionConfiguration) => {
     const { columnsToBeRemoved, cellHeaders } = extractionConfiguration
+    const normalizedCellHeaders = cellHeaders.map(header => header.trim().toLowerCase())
```

Comment:
The `.trim()` has changed the behaviour of this method, and I think caused you to change the tests to match the change.

I don't think the change here is right.

The metric seems to have a `' Habitat type'` column with a preceding whitespace. Before this method would have removed that column, but now it doesn't.

I don't think the issue happens with the config files use in the main code base as they have an `endCell` and I suspect `' Habitat type'` is a hidden column somewhere to the right of the `endCell`.

Preceding and trailing whitespace can already be handled by the `performSubstitution` method if needed.

I think you should revert to the original behaviour of this method.

Reply:
i have removed this change and update the extraction config